packages/venia-concept/venia-upward.yml

```diff
@@ -63,6 +63,12 @@ resource:
         inline:
           model: genericObject
           template: './templates/generic-shell.mst'
+    - matches: request.url.pathname
+      pattern: '/search.html'
+      use:
+        inline:
+          model: genericObject
+          template: './templates/generic-shell.mst'
```

Comment:
Just commenting for visibility on this change, I'm not sure if adding an explicit pathname as a test here is the best way to handle this. However, since the search component will not match on the urlResolver query, the page title and load would display "Page Not Found" if the search page is directly loaded without this.